### PR TITLE
Release 1.0.0-rc1

### PR DESCRIPTION
**Pull request by mayankshah1607**

### CODE REVIEW

File: sources/metadata/everest/1.0.0-rc1.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+version: 1.0.0-rc1
+supported:
+  cli: '>= 1.0.0'
```
Comment (mayankshah1607) [inline sources/metadata/everest/1.0.0-rc1.yaml:3]: It's okay to exclude the metadata tag here, since the CLI checks only the core version when verifying

**Comment by recharte** [comment]
This PR may also remove the `sources/metadata/everest/0.11.0-rc1.yaml` file, we won't need it anymore after the `1.0.0-rc1`

**Comment by mayankshah1607** [comment]
@recharte thanks, done!

**Review by recharte — APPROVED** [review]
Approving but let's wait for the RC to be built before merging

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by oksana-grishchenko — APPROVED** [review]

**Pull request merged**